src/main/java/com/google/api/generator/gapic/composer/ServiceClientCommentComposer.java

```diff
@@ -103,7 +108,12 @@
           "Returns the OperationsClient that can be used to query the status of a long-running"
               + " operation returned by another API method call.");
 
-  static List<CommentStatement> createClassHeaderComments(Service service) {
+  static List<CommentStatement> createClassHeaderComments(
+      Service service, Map<String, TypeNode> types) {
```

Comment:
Can we plumb the settings and client types parameters up, so we  don't have to pass in the entire map?

Reply:
Sure